spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/notify/microsoft/teams/MicrosoftTeamsNotifier.java

```diff
@@ -0,0 +1,161 @@
+            "/master/spring-boot-admin-server-ui/core/img/platform-spring-boot.jpg");
+
+    /**
+     * Theme Color is the color of the accent on the message that appears in Microsoft Teams. Default is Spring Green
+     */
+    private String themeColor = "6db33f";
+
+    private RestTemplate restTemplate;
+
+    public MicrosoftTeamsNotifier() {
+        // Initialize the restTemplate using builder to allow for customization
+        RestTemplateBuilder restTemplateBuilder = new RestTemplateBuilder();
+        this.restTemplate = restTemplateBuilder.build();
+    }
+
+    public Message getDeregisteredMessage(Application app) {
+        Message message = getMessage(app, DE_REGISTERED_TITLE, String.format(DEREGISTER_ACTIVITY_SUBTITLE_PATTERN,
+                app.getName(),
+                app.getId()));
+
+        return message;
+    }
+
+    public  Message getRegisteredMessage(Application app) {
+        return getMessage(app, REGISTERED_TITLE, String.format(REGISTER_ACTIVITY_SUBTITLE_PATTERN,
+                app.getName(),
+                app.getId()));
+    }
+
+    public Message getStatusChangedMessage(Application app, StatusInfo from, StatusInfo to) {
+        return getMessage(app, STATUS_CHANGED_TITLE, String.format(STATUS_ACTIVITY_SUBTITLE_PATTERN,
+                app.getName(),
+                app.getId(),
+                from.getStatus(),
+                to.getStatus()));
+    }
+
+    public Message getMessage(Application app, String registeredTitle, String activitySubtitle) {
+        Message message = new Message();
+
+        message.setTitle(registeredTitle);
+        message.setSummary(MESSAGE_SUMMARY);
+        message.setThemeColor(themeColor);
+
+        Section section = new Section();
+        section.setActivityTitle(app.getName());
+        if(imageUrl != null) {
+            section.setActivityImage(imageUrl.toString());
+        }
+        section.setActivitySubtitle(activitySubtitle);
+        section.getFacts().add(new Fact(STATUS_KEY, app.getStatusInfo().getStatus()));
+        section.getFacts().add(new Fact(SERVICE_URL_KEY, app.getServiceUrl()));
+        section.getFacts().add(new Fact(HEALTH_URL_KEY, app.getHealthUrl()));
+        section.getFacts().add(new Fact(MANAGEMENT_URL_KEY, app.getManagementUrl()));
+        section.getFacts().add(new Fact(SOURCE_KEY, app.getSource()));
+
+        message.getSections().add(section);
+
+        return message;
+    }
+
+    @Override
+    protected void doNotify(ClientApplicationEvent event) throws Exception {
+        if(event instanceof ClientApplicationRegisteredEvent) {
+            ClientApplicationRegisteredEvent registeredEvent = (ClientApplicationRegisteredEvent)event;
+            Message registeredMessage = getRegisteredMessage(registeredEvent.getApplication());
+
+            this.restTemplate.postForObject(webhookUrl, registeredMessage, Object.class);
```

Comment:
If you dont use the response-body please use `.postForObject(webhookUrl, registeredMessage, Void.class)`